conf.d/sequins.yaml.example

```diff
@@ -0,0 +1,11 @@
+# This check takes no init_config
+init_config:
```

Comment:
Maybe add an example of a commented out `min_collection_interval` here since you implied it would be a normal thing to adjust? (That way we also don't forget, lol)